clients/admin-ui/src/features/dataset/ApproveClassification.tsx

```diff
@@ -44,39 +47,53 @@ const ApproveClassification = () => {
     const updatedDataset = getUpdatedDatasetFromClassifyDataset(
       dataset,
       classifyDataset,
-      classifyCollection?.name
+      classifyCollection?.name,
+      features.flags.datasetClassificationUpdates
```

Comment:
you could also, instead of passing the flag down, do

```
getUpdatedDatasetFromClassifyDataset(
      dataset,
      classifyDataset,
      classifyCollection?.name
      features.flags.datasetClassificationupdates ? classifyCollection?.name : undefined,
)
```

and then update `getUpdatedDatasetFromClassifyDataset` to only do the collection check if `activeCollection` exists? that might also be safer to save us from the case of collections without defined names potentially giving us problems

Reply:
Similar to above, I ended up a little confused as to what the best way to maintain some sort of traceability to this being a feature flag was vs some other boolean condition - open to doing whatever makes sense here tho!

Reply:
Thanks for the details on how to make this happen! Updated in [2f69721](https://github.com/ethyca/fides/pull/2335/commits/2f6972197d3519bdc9804bb4396572e6db495d74)